.github/workflows/test.yml

```diff
@@ -31,9 +31,6 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        # We claim to support from 5.6+ but WP Mock only supports 7.1+
-        # Also an issue with WP Mock not working on PHPUnit 9.5+, which PHP 7.3+ needs.
-        # php: [ '5.6', '7.0', '7.4', '8.0', '8.1' ]
         php: [ '7.1', '7.2' ]
```

Comment:
7.1 and 7.2 looks like a deprecated support, needs to be updated to 7.4 and 8.0?

Reply:
Yes, we should, but need to fix the PHPUnit test failure issue first as mentioned above.

Reply:
Done, https://github.com/10up/simple-local-avatars/pull/143/commits/c46a7337186d93b34c21358400f48b70fd8e8a5c